Regression/WarpX-tests.ini

```diff
@@ -1153,6 +1153,22 @@ stSuccessString = Passed
 doVis = 0
 tolerance = 1.e-14
 
+[LaserInjectionFromTXYEFile3D]
+buildDir = .
+inputFile = Examples/Modules/laser_injection_from_file/analysis3d.py
+aux1File = Examples/Modules/laser_injection_from_file/inputs.3d_test_txye
+customRunCmd = ./analysis3d.py
+dim = 3
+addToCompileString =
+restartTest = 0
+useMPI = 0
```

Comment:
With out test build matrix, we have to pay attention to how many compilations we need per build job. Is this test using `useMPI = 0` (i.e., require to build without MPI) on purpose? Note that it is totally fine, we have a build job for this case that we would love to populate, just want to check here. Same for the test above.

Reply:
No, there is no particular reason for  performing this test with `useMPI=0` . We are free to keep this option as is or to change it.